infra/gp-grafana-operator/templates/01-operator-grooup.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: Grafana.v1alpha1.integreatly.org,GrafanaDashboard.v1alpha1.integreatly.org,GrafanaDataSource.v1alpha1.integreatly.org,GrafanaNotificationChannel.v1alpha1.integreatly.org
```

Comment:
kann man vielleicht sogar weglasse. Ich glaub der Operator-Lifecycle-Manager setzt die Labels automatisch abhängig von den Subscriptions in der Gruppe. 
Damit hätte man weniger Wartungsaufwand, sollten sich die APIs des Operators ändern.